src/main/webapp/css/srp_style.css

```diff
@@ -43,3 +43,11 @@
   margin-top: 0;
   display: block;
 }
+
+.card-heading {
+  display: flex;
```

Comment:
Flexbox seems overkill. You just have two inline elements next to each other. If you're trying to do vertical alignment, use the `vertical-align` property on the image: https://developer.mozilla.org/en-US/docs/Web/CSS/vertical-align

Here, I think you want `vertical-align: bottom;`